### PR TITLE
SEO-175190 Xamarin Page with Redirect

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @Aishwarya-Ganesan,

Have changed Page with Redirect links.

Regards,
Yvone.

### CODE REVIEW

File: Xamarin/Backdrop-Page/Getting-Started.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ This section explains the steps required to configure the backdrop page.
 
 ## Assembly deployment
 
-After installing [Essential Studio for Xamarin](https://www.syncfusion.com/downloads/xamarin), you can find all the required assemblies in the installation folders, {Syncfusion Essential Studio Installed location} \Essential Studio\\{Version #}\Xamarin\lib.
+After installing [Essential Studio for Xamarin](https://www.syncfusion.com/account/manage-trials/downloads), you can find all the required assemblies in the installation folders, {Syncfusion Essential Studio Installed location} \Essential Studio\\{Version #}\Xamarin\lib.
```
Comment (MallikaRK) [inline Xamarin/Backdrop-Page/Getting-Started.md:16]: revert this changes, Check and correct in all places.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: Xamarin/Backdrop-Page/Getting-Started.md (hunk 1 above)
Comment (Yvone-Atieno) [reply Xamarin/Backdrop-Page/Getting-Started.md:16]: changes reverted

**Review by MallikaRK — APPROVED** [review]
Approved

**Pull request merged**